chooloolib/src/main/java/com/chooloo/www/chooloolib/model/ListData.kt

```diff
@@ -62,7 +70,7 @@ data class ListData<DataType>(
 
         fun fromPhones(phones: List<PhoneAccount>): ListData<PhoneAccount> {
```

Comment:
You can pass `context` to `fromPhones()` and use that, using a static instance of the app and in a dedicated object inside this data class is not a recommended behaviour :)